### PR TITLE
fs2bs: remove a legacy fact

### PR DESCRIPTION
**Pull request by guits**
since cf7345f143148a6be2d71954f829a8f7fe11ab22, we don't need to set
this fact anymore.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test centos-non_container-filestore_to_bluestore

**Comment by guits** [comment]
jenkins test centos-container-filestore_to_bluestore

**Review by dsavineau — APPROVED** [review]

**Comment by guits** [comment]
jenkins test centos-container-filestore_to_bluestore

**Comment by guits** [comment]
jenkins test centos-non_container-filestore_to_bluestore

File: infrastructure-playbooks/filestore-to-bluestore.yml [hunk 1]
```diff
@@ -33,13 +33,8 @@
...
         tasks_from: container_binary.yml
 
-    - name: set_fact container_run_cmd, container_exec_cmd
-      set_fact:
-        container_run_cmd: "{{ container_binary + ' run --rm --privileged=true --net=host --pid=host --ipc=host -v /dev:/dev -v /etc/ceph:/etc/ceph -v /var/lib/ceph:/var/lib/ceph -v /var/run:/var/run --entrypoint=' if containerized_deployment | bool else '' }}ceph-volume {{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment | bool else '' }}"
-        container_exec_cmd: "{{ container_binary + ' exec ceph-mon-' + hostvars[groups[mon_group_name][0]]['ansible_hostname'] if containerized_deployment | bool else '' }}"
-
     - name: get ceph osd tree data
-      command: "{{ container_exec_cmd }} ceph --cluster {{ cluster }} osd tree -f json"
+      command: "{{ container_binary + ' exec ceph-mon-' + hostvars[groups[mon_group_name][0]]['ansible_hostname'] if containerized_deployment | bool else '' }} ceph osd tree -f json"
```
Comment (dsavineau) [inline infrastructure-playbooks/filestore-to-bluestore.yml:37]: the `{{ cluster }}` variable is missing

**Review by dsavineau — CHANGES_REQUESTED** [review]

**Comment by guits** [comment]
jenkins test centos-non_container-filestore_to_bluestore

**Comment by guits** [comment]
jenkins test centos-container-filestore_to_bluestore

File: infrastructure-playbooks/filestore-to-bluestore.yml [hunk 2]
```diff
@@ -33,13 +33,8 @@
...
         tasks_from: container_binary.yml
 
-    - name: set_fact container_run_cmd, container_exec_cmd
-      set_fact:
-        container_run_cmd: "{{ container_binary + ' run --rm --privileged=true --net=host --pid=host --ipc=host -v /dev:/dev -v /etc/ceph:/etc/ceph -v /var/lib/ceph:/var/lib/ceph -v /var/run:/var/run --entrypoint=' if containerized_deployment | bool else '' }}ceph-volume {{ ceph_docker_registry + '/' + ceph_docker_image + ':' + ceph_docker_image_tag if containerized_deployment | bool else '' }}"
-        container_exec_cmd: "{{ container_binary + ' exec ceph-mon-' + hostvars[groups[mon_group_name][0]]['ansible_hostname'] if containerized_deployment | bool else '' }}"
-
     - name: get ceph osd tree data
-      command: "{{ container_exec_cmd }} ceph --cluster {{ cluster }} osd tree -f json"
+      command: "{{ container_binary + ' exec ceph-mon-' + hostvars[groups[mon_group_name][0]]['ansible_hostname'] if containerized_deployment | bool else '' }} --cluster {{ ceph }} ceph osd tree -f json"
```
Comment (dsavineau) [inline infrastructure-playbooks/filestore-to-bluestore.yml:37]: almost correct :D you can do it maggle

**Review by dsavineau — CHANGES_REQUESTED** [review]

**Comment by guits** [comment]
jenkins test centos-non_container-filestore_to_bluestore

**Comment by guits** [comment]
jenkins test centos-container-filestore_to_bluestore

**Review by dsavineau — CHANGES_REQUESTED** [review]
I just saw that `container_exec_cmd` is still used later in the playbook

**Comment by guits** [comment]
jenkins test centos-container-filestore_to_bluestore

**Comment by guits** [comment]
jenkins test centos-non_container-filestore_to_bluestore

**Review by dsavineau — APPROVED** [review]

**Comment by dsavineau** [comment]
jenkins test centos-non_container-filestore_to_bluestore

**Comment by guits** [comment]
jenkins test centos-non_container-filestore_to_bluestore

**Comment by guits** [comment]
jenkins test centos-non_container-filestore_to_bluestore

**Pull request merged**